_includes/assets/css/inline.css

```diff
@@ -0,0 +1,187 @@
+:root {
```

Comment:
Should we delete this file now that we're using sass?

Reply:
Same as above. I'd like to keep just for now with a view to deleting. I'm sometimes looking at for reference for how 11ty theme compiles.